pandas/core/computation/expressions.py

```diff
@@ -223,7 +223,7 @@ def evaluate(op, a, b, use_numexpr: bool = True):
     use_numexpr : bool, default True
         Whether to try to use numexpr.
     """
-    op_str = _op_str_mapping.get(op, None)
+    op_str = _op_str_mapping[op]
     if op_str is not None:
```

Comment:
Isnt this always true, so you can drop the if statement now?

Reply:
it is not always true, some entries in _op_str_mapping give None